README.md
# PICInfo

source/changes.adoc
== Changes
*Formal Release*

{empty} +
[cols="^1,^1", options="header",width="60%"]
|===
|*Reference*
|*Time Stamp*

|ASCIIDOCs rendered
|{localdate} +
{localtime}

|Master ToC information
|{docdate} +
{doctime}

|===

{empty} +

*Changes in this release*::

First release.


output/web/picinfo/search/index-1.js
var indexerLanguage="en";
//Auto generated index for searching by xsl-webhelpindexer for DocBook Webhelp.# Kasun Gajasinghe, University of Moratuwa
w["-"]="0*1,2*2,3*2,4*2,5*2,6*2,7*2,8*2,9*3";
w["."]="3*1,5*1";
w["0"]="1*1";
w["02"]="2*1";
w["03"]="2*2";
w["05"]="2*1";
w["05:07:02"]="2*1";
w["07"]="2*1";
w["11"]="2*1";
w["11:23:51"]="2*1";
w["12"]="2*1";
w["2020"]="0*1,2*1";
w["2020-12-21"]="2*1";
w["2022"]="2*1";
w["2022-03-03"]="2*1";
w["21"]="2*1";
w["23"]="2*1";
w["51"]="2*1";
w["7"]="1*1";
w["8"]="6*2";
w["8-bit"]="6*2";
w["access"]="9*2";
w["address"]="1*2,8*2";
w["after"]="5*1";
w["all"]="3*1,5*1,9*1";
w["also"]="6*1";
w["ani"]="6*1,9*1";
w["applic"]="1*1,6*14,7*1,8*1,9*1";
w["application."]="6*1,9*3";
w["area"]="9*1";
w["asciidoc"]="2*1";
w["ask"]="3*1,5*1";
w["avail"]="3*1,5*1";
w["avr"]="6*1";
w["avrinfo"]="6*1";
w["avrinfo."]="6*1";
w["basic"]="0*1,1*1,3*4,4*1,5*7,6*4,7*1,8*1,9*6";
w["below"]="6*1,9*1";
w["below:"]="6*1";
w["bit"]="0*1,1*50,5*2,6*3,7*1";
w["bit_valu"]="1*1";
w["bits."]="7*1";
w["browser"]="9*2";
w["browser."]="9*2";
w["buffer"]="9*1";
w["build"]="6*5";
w["button"]="4*1,9*7";
w["button."]="4*1";
w["button:"]="9*5";
w["chang"]="0*1,2*51";
w["chip"]="3*1";
w["chipdata"]="0*1,3*46,6*3";
w["chipdata.def"]="6*1";
w["clear"]="4*3,5*1";
w["clear:"]="4*1";
w["clipboard"]="9*1";
w["clipboard.  "]="9*1";
w["code"]="6*1";
w["colum"]="1*1,3*1,5*1,8*1";
w["compiler."]="6*1";
w["complet"]="5*1";
w["completed..  "]="5*1";
w["config"]="7*1,9*13";
w["configu"]="9*5";
w["configur"]="6*2,7*4,9*5";
w["configuration.  "]="9*1";
w["conigura"]="6*1";
w["constant"]="3*5";
w["constant.  "]="3*1";
w["contain"]="6*1";
w["content"]="0*5,2*1";
w["copi"]="9*8";
w["cow"]="0*1,3*3,5*6,6*4,9*6";
w["creat"]="6*1";
w["cut"]="9*2";
w["dashboard"]="9*6";
w["data"]="1*1,3*1,5*1,7*1,8*1";
w["datasheet"]="1*1,3*1,5*1,7*1,8*1,9*3";
w["datasheets."]="9*1";
w["decim"]="8*1";
w["def"]="6*1";
w["default"]="6*4,9*2";
w["descript"]="3*1";
w["detail"]="1*1,2*1,3*1,5*1,6*1,7*1,8*1";
w["develop"]="0*1,6*1";
w["dialog"]="1*1,3*1,4*2,5*1,6*1,7*1,8*1,9*2";
w["dialog.  "]="9*1";
w["dialog:"]="4*1";
w["dialogs:"]="9*1";
w["document"]="0*51,2*1,3*1,4*1,5*1,6*1,7*1,8*1,9*1";
w["doe"]="4*1";
w["download"]="6*1";
w["drop"]="9*1";
w["dropdown"]="4*1,9*1";
w["dropdown.  "]="4*1";
w["edit"]="6*1,9*3";
w["editor."]="9*1";



output/web/picinfo/search/index-2.js
//Auto generated index for searching by xsl-webhelpindexer for DocBook Webhelp.# Kasun Gajasinghe, University of Moratuwa
w["edits."]="9*1";
w["either"]="9*1";
w["enabl"]="1*1,3*1,4*1,5*2,6*1,7*1,8*1,9*1";
w["enjoy"]="6*1";
w["enjoy.  "]="6*1";
w["errata"]="1*1,3*1,5*1,7*1,8*1,9*2";
w["exampl"]="1*5,3*5,5*5,7*5,8*5";
w["exist"]="4*1";
w["explain"]="3*1";
w["expos"]="7*1";
w["express"]="1*1";
w["field"]="7*2";
w["file"]="2*1,9*2";
w["file."]="2*1,6*1";
w["filter"]="0*1,4*50,9*2";
w["filter.  "]="4*1";
w["first"]="2*1,6*5,7*1";
w["folder"]="2*1,6*4";
w["folder."]="6*3";
w["follow"]="6*1,9*1";
w["following:"]="9*1";
w["formal"]="2*5";
w["forum"]="3*4,5*4";
w["four"]="1*1";
w["free"]="6*1";
w["function"]="6*1";
w["gcb"]="9*2";
w["gcbasic"]="3*3,5*3";
w["generat"]="6*1";
w["gmt"]="2*2";
w["great"]="0*1,3*3,5*6,6*4,9*6";
w["help"]="2*1";
w["hex"]="8*1";
w["http"]="3*1,5*1";
w["http:"]="3*2,5*2";
w["ide"]="9*1";
w["ide."]="9*1";
w["idebutton"]="9*1";
w["idebutton:"]="9*1";
w["if"]="1*1,7*1,8*1";
w["includ"]="6*1";
w["inform"]="2*1";
w["initi"]="6*1";
w["initialis"]="6*1";
w["insert"]="6*1";
w["instal"]="6*1";
w["interfac"]="6*1,9*1";
w["interrupt"]="0*1,5*54,6*1";
w["interrupts.  "]="6*1";
w["into"]="6*1,9*1";
w["introduc"]="0*1,6*46";
w["introduct"]="1*5,3*5,4*5,5*5,7*5,8*5,9*5";
w["introduction:"]="1*5,3*5,4*5,5*5,7*5,8*5,9*5";
w["intuit"]="6*1";
w["item"]="9*1";
w["key"]="9*1";
w["launch"]="9*3";
w["like"]="6*1";
w["linux"]="6*1";
w["load"]="6*1";
w["maco"]="6*1";
w["macos."]="6*1";
w["made"]="9*1";
w["main"]="9*1";
w["mark"]="9*1";
w["master"]="2*1";
w["match"]="4*1";
w["match.  "]="4*1";
w["menu"]="9*1";
w["menu:"]="9*1";
w["message."]="6*1";
w["method"]="9*1";
w["methods.  "]="9*1";
w["microchip"]="6*1,9*3";
w["microcont"]="6*1";
w["microcontrol"]="1*3,3*3,4*2,5*3,6*3,7*3,8*3,9*16";
w["microcontroller."]="1*2,3*1,5*1,6*1,7*2,8*2,9*1";
w["modifi"]="6*1";
w["must"]="6*5";
w["name"]="1*1,3*1,5*1,8*1";
w["net"]="3*1,5*1";
w["number"]="4*1";
w["ok"]="6*1";
w["onli"]="6*1,9*2";
w["only.  select"]="9*2";
w["open"]="6*1,9*3";
w["oper"]="6*11,9*2";
w["option"]="0*1,7*49,9*8";
w["other"]="7*1,9*1";
w["p"]="3*1";
w["page"]="9*1";
w["paramet"]="6*1,7*2";
w["past"]="9*3";
w["pdf"]="9*2";
w["permit"]="9*1";
w["pic"]="4*1,6*2,9*6";
w["picinfo"]="0*52,2*1,3*1,4*1,5*1,6*51,7*1,8*1,9*1";



output/web/picinfo/search/htmlFileInfoList.js
var doStem = true;
//List of indexed files.
fl = new Array();
fl["0"]= "index.html";
fl["1"]= "_bits_tab.html";
fl["2"]= "_changes.html";
fl["3"]= "_chipdata_tab.html";
fl["4"]= "_filters.html";
fl["5"]= "_interrupts_tab.html";
fl["6"]= "_introducing_picinfo.html";
fl["7"]= "_options_tab.html";
fl["8"]= "_registers_tab.html";
fl["9"]= "_select_tab.html";
fil = new Array();
fil["0"]= "index.html@@@PICInfo documentation@@@null";
fil["1"]= "_bits_tab.html@@@Bits Tab@@@null";
fil["2"]= "_changes.html@@@Changes@@@null";
fil["3"]= "_chipdata_tab.html@@@ChipData Tab@@@null";
fil["4"]= "_filters.html@@@Filters@@@null";
fil["5"]= "_interrupts_tab.html@@@Interrupts Tab@@@null";
fil["6"]= "_introducing_picinfo.html@@@Introducing PICInfo@@@null";
fil["7"]= "_options_tab.html@@@Options Tab@@@null";
fil["8"]= "_registers_tab.html@@@Registers Tab@@@null";
fil["9"]= "_select_tab.html@@@Select Tab@@@null";


output/web/picinfo/search/index-3.js
//Auto generated index for searching by xsl-webhelpindexer for DocBook Webhelp.# Kasun Gajasinghe, University of Moratuwa
w["pictur"]="4*1,6*1,9*1";
w["pleas"]="1*1,2*1,3*1,5*1,7*1,8*1";
w["pps"]="9*3";
w["pps.  select"]="9*1";
w["pref"]="9*1";
w["prefer"]="6*1,9*3";
w["prefix"]="3*1";
w["program"]="5*1";
w["project"]="3*3,5*3,6*3";
w["project.  "]="6*1";
w["provid"]="6*1";
w["question"]="3*1,5*1";
w["reduc"]="4*1";
w["refer"]="1*1,2*6,3*1,5*1,7*1,8*1";
w["regex"]="4*1";
w["regis"]="1*1";
w["regiser.bit_value."]="1*1";
w["regist"]="0*1,1*2,6*2,8*51";
w["register."]="8*1";
w["relat"]="1*1";
w["releas"]="2*11,6*2";
w["release."]="2*1,6*1";
w["remov"]="9*1";
w["render"]="2*1";
w["requir"]="6*1";
w["reset"]="9*8";
w["rogram"]="3*1";
w["roller"]="6*1";
w["rollers."]="6*1";
w["routin"]="5*1";
w["search"]="1*7,3*7,4*2,5*7,7*7,8*7,9*2";
w["search:"]="1*1,3*1,5*1,7*1,8*1,9*1";
w["searches.  "]="4*1";
w["section"]="2*1";
w["see"]="3*1,5*1,6*1,9*1";
w["select"]="0*1,4*2,6*4,7*1,9*61";
w["selected."]="7*1";
w["selections."]="9*1";
w["set"]="4*1,6*1,9*2";
w["share."]="6*1";
w["show"]="1*2,3*2,4*2,5*2,6*3,7*1,8*2,9*1";
w["shown"]="1*1,3*1,5*1,7*1,8*1,9*1";
w["shown."]="1*1,3*1,5*1,7*1,8*1";
w["softwar"]="6*3";
w["software."]="6*1";
w["sourc"]="6*2";
w["sourceforg"]="3*1,5*1";
w["sourceforge.net"]="3*2,5*2";
w["specif"]="1*2,3*1,5*1,6*6,7*3,8*2,9*4";
w["stamp"]="2*5";
w["standard"]="2*2";
w["start"]="6*1";
w["sub"]="2*1";
w["sub-sect"]="2*1";
w["support"]="4*1,6*10,9*1";
w["synwrit"]="9*1";
w["system"]="6*11,9*1";
w["systems.  "]="6*1";
w["t"]="7*1";
w["tab"]="0*6,1*51,3*52,4*1,5*52,7*52,8*52,9*49";
w["tabl"]="0*5";
w["team"]="0*1";
w["term"]="6*1";
w["tha"]="7*1";
w["that"]="6*1,7*1,9*1";
w["them"]="9*1";
w["then"]="6*1";
w["these"]="9*1";
w["three"]="3*1,5*1,8*1";
w["time"]="2*7";
w["tion"]="6*1";
w["toc"]="2*1";
w["tool"]="9*2";
w["two"]="9*1";
w["up"]="6*1";
w["use"]="2*1,4*1,5*3,6*6,7*1,9*2";
w["user"]="3*1,5*1";
w["util"]="9*7";
w["utilities."]="9*1";
w["valid"]="6*1";
w["valu"]="3*1";
w["view."]="2*1";
w["web"]="9*3";
w["webpag"]="9*1";
w["welcom"]="6*1";
w["when"]="5*1";
w["wildcard"]="4*1";
w["will"]="5*2,6*2,9*4";
w["window"]="6*2,9*3";
w["within"]="3*1,5*1,9*1";
w["xojo"]="6*1";
w[" "]="0*3,1*4,2*4,3*4,4*4,5*4,6*4,7*4,8*4,9*4";
w["  "]="3*1,4*3,5*1,6*5,9*5";
w["  select"]="9*3";

